src/value.rs

```diff
@@ -1177,34 +1132,6 @@ impl Values {
 mod tests {
     use super::*;
 
-    #[test]
-    fn test_escape_1() {
-        let test = r#" "abc" "#;
-        assert_eq!(escape_string(test), r#" \"abc\" "#.to_owned());
-        assert_eq!(unescape_string(escape_string(test).as_str()), test);
-    }
-
-    #[test]
-    fn test_escape_2() {
-        let test = "a\nb\tc";
-        assert_eq!(escape_string(test), "a\\nb\\tc".to_owned());
-        assert_eq!(unescape_string(escape_string(test).as_str()), test);
-    }
-
-    #[test]
-    fn test_escape_3() {
-        let test = "a\\b";
-        assert_eq!(escape_string(test), "a\\\\b".to_owned());
-        assert_eq!(unescape_string(escape_string(test).as_str()), test);
-    }
-
-    #[test]
-    fn test_escape_4() {
-        let test = "a\"b";
-        assert_eq!(escape_string(test), "a\\\"b".to_owned());
-        assert_eq!(unescape_string(escape_string(test).as_str()), test);
-    }
-
```

Comment:
Hey @ikrivosheev, could we add back these test cases in query test?

Reply:
@billy1624 I cannot find query test.

Reply:
Agree with Billy we'd better not delete the test cases (actually we might want more)

Reply:
@tyt2y3 @billy1624, done, I divided the tests for each backend.

Reply:
Thanks! @ikrivosheev